addons/account/wizard/account_move_reversal.py

```diff
@@ -96,8 +96,12 @@ def _compute_from_moves(self):
 
     def _prepare_default_reversal(self, move):
         reverse_date = self.date if self.date_mode == 'custom' else move.date
+        partner_bank_id = self.env['res.partner.bank'].search([
```

Comment:
That's a `search` in a loop (https://github.com/odoo/odoo/pull/173640/files#diff-1bfd24cca21724d67d0b85aaadaec5b89e5d2c17a0766d33cccdb0840d54356dR123) :(
Can't we find a better solution (without destroying the stable policy by changing the method signature)?

Reply:
I have moved the search out of the loop :+1: